packages/dx-grid-core/src/plugins/virtual-table-state/test-utils.ts

```diff
@@ -0,0 +1,12 @@
+import { Interval } from '../../types';
+
+export const createInterval = (start: number, end: number) => ({ start, end });
```

Comment:
It will not deduce that return type is `Inverval`, will it?

```js
export const createInterval = (start: number, end: number): Interval => ({ start, end });
```